packages/indexer-client/src/IndexerBaseClient.ts

```diff
@@ -781,7 +781,7 @@ export class IndexerBaseClient {
   ): Promise<GetIndexerBlitzPointsLeaderboardResponse> {
     const baseResponse = await this.query('blitz_points_leaderboard', {
       limit: params.limit,
-      start: params.startCursor,
+      start: Number(params.startCursor),
```

Comment:
This was needed, else server throws an error (not sure if you've tested via the sanity test)